sdk/core/azure-core-xml/inc/azure/core/xml.hpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @brief Includes all public headers from Azure Core Xml SDK library.
+ *
+ */
+
+#pragma once
+
+#include "azure/core/xml/dll_import_export.hpp"
+#include "azure/core/xml/rtti.hpp"
+#include "azure/core/xml/xml_wrapper.hpp"
```

Comment:
Similar to `azure-core-opentelemetry`, this library is not providing any public API, only the internal one, so it should have no all-inclusive header.

And if it had, all-inclusive headers should not include internal headers (`internal/xml_wrapper.hpp`).

And if we had this file, it cannot be in `core/xml.hpp` - that would create a collision with the core package. This file should have been in `inc/core/xml/xml.hpp` , similar to how `inc/core/amqp/amqp.hpp` does.

But just delete this file.